src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.2.0-preview",
+  "version": "1.3.0-preview",
```

Comment:
@DheerendraRathor all project.json, global.json and *.xproj files should be deleted. We no longer need them.